src/main/java/at/klausbetz/provider/AppleIdentityProviderConfig.java

```diff
@@ -44,4 +46,9 @@ public void setP8Content(String p8Content) {
     public boolean isDisableUserInfoService() {
         return true;
     }
+
+    @Override
+    public String getDisplayIconClasses() {
+        return DISPLAY_ICON_CLASSES;
```

Comment:
This method is not called right now, as only IdentityProviderModel is intantiated and not AppleIdentityProviderConfig.